### ISSUE TITLE
Bulk Upload Processing Time

### ISSUE THREAD

**Issue by JRWinWin** [winwinwiki/winwin-frontend#479]
I was able to successfully upload a bulk file of organizations!
The file that was uploaded contained about 133k records and it was 15.7 MB. 
I timed the upload process and it took 12 hours and 45 minutes to complete. 
We have stated that we would like to be able to upload files that contain upwards of 1 million records. At this rate, the upload process for 1 million records would take approximately 100 hours, or 4 days. 
We will be testing any potential conflicts while the upload process is occurring today. 
Is there any way to expedite the bulk upload process? If the bulk upload speed remains the same, it could have detrimental effects on our ability to scale.

**Comment by arvindkhatik012** [comment]
@JRWinWin @BethRR This might be happening due to that the postgres instance on dev server is currently hosted in us-east-1 region and our application is hosted in us-west-2 region, I think this is happening due to network lag because we have tried to upload 133k organisations in our local machine postgres db and it only took 90seconds to load.

So for a workaround we'll try to upload the same file in staging environment as all the instances are hosted in us-west-2 region and for few days we'll do the analysis on server capacity and accordingly we'll increase the server loading capacity to avoid extra charges by amazon.

**Comment by JRWinWin** [comment]
@arvindkhatik012 We are still experiencing this issue. I attempted another upload of 133k and it took around 12 hours again to complete. What is the progress on this issue?

**Comment by arvindkhatik012** [comment]
@BethRR As I explained you the reason in my earlier comments,the issue is happening due to network lag because we have already tried to load 133k organisations and it took only 90seconds.

We will upload the same organisations in our new staging environment as all the environment are hosted in same region i.e. us-west-2 and do the analysis of processing time and we'll come with workaround whether we need to upgrade our server efficiency or not?
